### DISCUSSION TITLE
Fibonacci Series

### DISCUSSION THREAD

**Discussion by Mohammed-Aljazzar** [community/community#65342] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The Fibonacci series is a sequence of numbers where each number is the sum of the two preceding ones. In this problem, you need to generate the Fibonacci series up to a given number of terms and display the resulting series.

These problems are often used as introductory exercises for programming beginners due to their simplicity and applicability in practicing core programming concepts like loops, conditionals, string manipulation, and sequence generation. Sharing your solutions to these problems on GitHub can be a great way to demonstrate your coding skills and help others learn these fundamental concepts.

**Comment by salahsaeed19** [comment]
def fibonacci(n):
    fib_series = [0, 1]
    while len(fib_series) < n:
        next_num = fib_series[-1] + fib_series[-2]
        fib_series.append(next_num)
    return fib_series

num_terms = int(input("Enter the number of terms: "))
fib_series = fibonacci(num_terms)
print("Fibonacci Series:", fib_series)

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
